using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -797,28 +900,32 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 {% tab title="React Native" %}
 #### JavaScript
 
-### setAdvertisingIdentifier
+### setAdvertisingIdentifier<a id="setAdvertisingIdentifier-js"></a>
 
 ```jsx
 ACPCore.setAdvertisingIdentifier("adID");
 ```
+o *adID* is a String that provides developers with a simple, standard system to continue to track the Ads through their apps	
+
 {% endtab %}
 {% endtabs %}
 
-## Set the push identifier
+## Set the push identifier<a id="setPushIdentifierTitle"></a>
 
-This API sets the device token for push notifications in the SDK. If the current SDK privacy status is `optedout`, the push identifier is not set.
+This `setPushIdentifier` API sets the device token for push notifications in the SDK. If the current SDK privacy status is `optedout`, the push identifier is not set.
```

Comment:
Reword to:
"This API sets the device token for push notifications in the SDK. If the current SDK privacy status is `optedout`, the push identifier is not set."